lib/reporters/cli/index.js

```diff
@@ -13,7 +13,8 @@ var _ = require('lodash'),
     DOT = '.',
     E = '',
 
-    PostmanCLIReporter;
```

Comment:
In the future, let's place all such variable additions above, so the diff is cleaner 😄